packages/create-webpack-app/src/generators/init/vue.ts

```diff
@@ -0,0 +1,250 @@
+// Cspell:ignore plopfile, plopfile.js
```

Comment:
why is cspell ignored?

Reply:
cspell was not recognising these words
so early on instead of tampering cspell.json I thought i should use this.

but now it seems this is in a lot of files after that major refactor.

I think I should add this word to dictionary as well?
What do you think?